tracer/src/Datadog.Trace/Logging/Internal/DatadogLogging.cs

```diff
@@ -44,6 +44,8 @@ static DatadogLogging()
 
                 var domainMetadata = DomainMetadata.Instance;
                 SharedLogger = DatadogLoggingFactory.CreateFromConfiguration(in config, domainMetadata) ?? SharedLogger;
+
+                LibDatadog.Logger.Instance.Enable(config, domainMetadata);
```

Comment:
I don't think this belongs here 🤔 It's fundamentally got nothing to do with _our_ logging system really, so it should probably be configured in the `TracerManagerFactory` just _after_ we configure the datapipeline (so that, for example, if we fail to configure it, we don't call this method, or if we _reconfigure_, we update as required)...